### ISSUE TITLE
screen reader read 'midnight' when reading out 00:00 instead of reading it as a timestamp

### ISSUE THREAD

**Issue by joshjm** [paypal/accessible-html5-video-player#138]
The windows screen reader reads timestamps as a time of day rather than a time since the start of the video.

This should be updated to not confuse users who use screen readers.

**Comment by devchristina** [comment]
![image](https://github.com/paypal/accessible-html5-video-player/assets/84518758/ad675d25-165c-469d-9ff0-ceaf0b11b79f)
